.circleci/config.yml

```diff
@@ -154,11 +152,9 @@ references:
               set -o xtrace
               colcon test \
                 --packages-select ${TEST_PACKAGES} \
```

Comment:
This line wrap will have to be removed if you remove the line below it, so that the test command may be invoked.